### ISSUE TITLE
Insertion Sort to Finish Off Radix Sort?

### ISSUE THREAD

**Issue by ParadaCarleton** [xiaodaigh/SortingLab.jl#17]
For small arrays, insertion sort is more efficient, so handing off a mostly-sorted array to an insertion sort should give speedups. From what I can see, this isn't implemented yet (although I might have missed it).

**Comment by xiaodaigh** [comment]
No it isn't. I think for smaller arrays, like size 12. That is implemented in base, so not that hard to copy the code over

**Comment by ParadaCarleton** [comment]
> No it isn't. I think for smaller arrays, like size 12. That is implemented in base, so not that hard to copy the code over

I don't think it even needs to be copied, you could just hand off to the base `sort!()` function. 

I think an array size of 32 or 16 is more likely to be optimal, but this should probably be benchmarked.

**Comment by xiaodaigh** [comment]
> I think an array size of 32 or 16 is more likely to be optimal, but this should probably be benchmarked.

The size just copies from base is fine
